delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resolvers/ResolverFields.scala

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.resolvers
+
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+
+/**
+  * Fields for a resolver
+  * @param id                id of the resolver
+  * @param priority          resolution priority when attempting to find a resource
+  * @param value             additional fields to configure the resolver
+  */
+final case class ResolverFields(id: Option[Iri], priority: Priority, value: ResolverValue)
```

Comment:
So now the "input" ADT and the Events ADT share the same object `ResolverValue`. It is fine for me, but we have to be especially careful then with serialization in this case. 
It is better when we have different ones, so we can have separate serializations, but I understand that here it is a big overhead to pay

Reply:
For ResolverFields, we have to implement a custom deserializer anyway.
For serializing the Resolver, a custom serializer will also be needed so as not to break the api.
It should help.